### PR TITLE
Undoes some VOREStation promethean nerfs

### LINKED ISSUE #3328: Promethean limbs come off incredibly easily.
#### Brief description of the issue
Promethean limbs come off INCREDIBLY easily, like. A single punch easily

#### What you expected to happen
Not to lose my limbs from someones most basic attack when they are mildly upset.

#### What actually happened
I accidently punched my head off while eating, and I had my limbs taken off with great ease and was generally made a useless prommie nugget until medical injected me with nutriment to the point I could regen.

#### Steps to reproduce
- (Step 1) Play a Promethean.
- (Step 2) Anger someone enough so that they hit you, or just have them jokingly punch you on the arm.
- (Step 3) Lose your limb and have to spend time and nutrition fixing it, and if its a bar fight possibly die because they took your arms off.

#### Code Revision
- Server revision: release - e467af8325bc7ab993a334c960db8351e8716a9b 
 
#### Anything else you may wish to add:
As mildly amusing as it is to easily become the Black Knight from Monty Python, it makes a few jobs rather untenable due to the fact that you might be dealing with accidents that deal more then 5 damage to your limbs, such as working in Xenobiology with anything other then slimes, security in general, engineering if things are really hairy, and anyone on a expedition if things go south.

### LINKED ISSUE #4795: Prometheans are pitifully weak, and Polaris wants to nerf them more
#### Brief description of the issue

**Primary Issues:**

- Current limb health is anywhere from 20-30. I've demonstrated extensively that Prommies can punch off their own heads/limbs in 2 hits, which must then be regenerated.

- Regeneration is time consuming, which is fine, but incredibly costly on the nutrition levels, requiring Prommies to be fully fed or above. On it's own, this would be a minor annoyance, but...

- Combined with a higher hunger speed and the fact that if regen is interrupted, the nutrition is still used, it becomes possible to nugget Prometheans permanently by removing their legs, letting them try to regen, interrupting it, and then dragging around your new victim.

- Lacking a head makes it impossible to eat normally, which isn't major, but kind of embarassing for those with table manners. They have a drain method, but you need a consenting participant, or an already subdued victim.

- Promethean Regen (Active and Passive) is done almost ENTIRELY via nutrition levels. Polaris intends to nerf this, and even make it cause pain because ???

**Secondary Issues:**

- Prometheans can't push/walk through anything that isn't another Prommie, a Teshari, or a simple animal. Previously circumvented by abusing Taur spaghetticode.

- The resistances they do have are completely offset by the weakness of their limbs, rendering the entire difference pointless.

- The worst nightmare of a prommie, welders, are widely available to any member of the crew. Not that this can really be fixed, but... it just compounds the existing issues.

**NITPICKING:**

- Prometheans can only speak 2 languages, one of which MUST BE GalCom

- The weakness to heat they have has never been adequately explained. XenoBio slimes don't seem to have a weakness to heat, but to cold.

#### What you expected to happen

To play a unique race with their own strengths and weaknesses, appearing genuinely alien in physiology, and for some characters, perhaps even mentality.

#### What actually happened

A downwards spiral of 'nerfs' and 'balancing' rendering Prommies easily killed by a stiff breeze. But hey, you're one of maybe 2 races unaffected by radiation, and can glow when exposed to it. That's nice, I guess.

#### Steps to reproduce
- Make a Promethean
- Punch your head 2 times
- Punch off a leg
- Congrats, you nuggeted yourself.

#### Code Revision
- current, 1fe04192c11e2ad7357512d5a873c44a4ea687d3

### PR DESCRIPTION
**Pull request by Heroman3003**
In the intial state PR only removes rebalance of their base species stats and lowered limb health. Those nerfs being undone don't have to be only changes made. Some of those perhaps should be kept and/or made less/more severe. This PR is moreso aimed at raising the attention towards the fact that combination of VOREStation nerfs to initial prometheans and Polaris nerfs that came in over time created a way-too-weak species. More changes than those suggested may need to be made, but proper discussion on just what needs to go and what needs to stay and what needs to stay in different way needs to be had, I believe.

In the final state, intent is to have PR that resolves #4795 and either reaffirms as intentional and kept even with other stuff or resolves #3328

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station/prometheans_vr.dm [hunk 1]
```diff
@@ -9,13 +9,7 @@
 		"Rapala", "Neaera", "Stok", "Farwa", "Sobaka",
 		"Wolpin", "Saru", "Sparra")
 
-	heal_rate = 0.2 //As of writing, original was 0.5 - Slows regen speed (bad)
-	hunger_factor = 0.1 //As of writing, original was 0.2 - Slows hunger rate (good)
-	siemens_coefficient = 1 //As of writing, original was 0.4 (bad)
-	active_regen_mult = 0.66 //As of writing, original was 1 (good)
-
 	color_mult = 1
-	mob_size = MOB_MEDIUM //As of writing, original was MOB_SMALL - Allows normal swapping (good)
```
Comment (Novacat) [inline code/modules/mob/living/carbon/human/species/station/prometheans_vr.dm]: This is not a nerf, but rather is there to make Vorestation features (especially size difference mechanics) work properly.

**Comment by Wickedtemp** [comment]
Might be a bit much to ask given how often Polaris seems to change prometheans, but if we could get a complete writeup of the species mechanics, that'd be extremely helpful.

And yeah, low limb health is just bad. It doesn't provide any balance, it's just there to be annoying.

**Comment by Screemonster** [comment]
They better not fucking melt on contact with water again

who the fuck thought firestacks was a good var to use for that anyway, THEY TOOK DAMAGE FROM BEING RAINED ON IN HARDSUITS

**Comment by Wickedtemp** [comment]
I'd be in favor of scrapping limb health as a whole, or making the limb health closer to 30-50. It's something that makes a bit of sense I guess, at the detriment of gameplay and hindrance of players, overall it's just bad design.

I'd also be in favor of removing their weakness to water. I don't care if prometheans can go out into the rain or in a pool without dying. I don't care if they can drink water without suffering any negative effect. 

The only real issue we had was that they'd win thunderdome matches rather easily, and yeah, in a set cage-match with few options to attack your enemy with, the promethean will probably win unless you have a weapon that causes burn damage. They have a x2 weakness to burn. Any burn weapon will fucking destroy a promethean. That's all anybody has ever had to do if they wanted to just objectively wreck any promethean at any point in time. 

They've been completely fine more or less since they were first added, nearly all of these changes are a result of a different server trying to make prometheans more 'combat-friendly' while also completely forgetting that all they needed to do to the promethean was jab it with acid or something and it'd just about instadie.

**Comment by Heroman3003** [comment]
VOREStation nerfs, imo, are the hardest, because of all limb health reductions. Upstream still has more 'normal' limb health which is for most organs more than twice what we have right now. So all the 'other' changes for 'combat' do make sense in context of that.

**Comment by Novacat** [comment]
I am pretty sure we already commented out Promethean weakness to water?

**Comment by Wickedtemp** [comment]
Prometheans have had so many changes that I stopped keeping track, so iunno.

**Comment by Novacat** [comment]
With all the nerfs prommies got, I think it is fair to undo the Virgo nerfs. Considering feedback for this change has been universally positive, going to merge this.

**Pull request merged**